src/main/java/com/synopsys/protecode/sc/jenkins/Scanner.java

```diff
@@ -226,9 +227,10 @@ private void addUploadResponse(PrintStream log, String name, HttpTypes.UploadRes
     // TODO: compare the sha1sum and send again if incorrect
     if (NO_ERROR.equals(error)) {
       results.add(new FileResult(name, response, zippingInUse));
+      console.logPure("Uploaded: " + name + ": " + "\n\t" + response.getResults().getReport_url());
```

Comment:
so not to make the console logging too simple. this just prints what is given. Of course later it can be prettied up